tests/CMakeLists.txt

```diff
@@ -4,6 +4,62 @@
 ##    in the file "license.txt" in the top-level Albany directory  //
 ##*****************************************************************//
 
+MESSAGE ("\n ***********************")
+MESSAGE (" *** Testing Options ***")
+MESSAGE (" ***********************\n")
+
+# Add option to enable performance tests. Results are machine-specific
+OPTION(ENABLE_PERFORMANCE_TESTS "Flag to turn on Performance Tests" OFF)
```

Comment:
I don't think this is used any longer. We moved the performance tests in another repo.